src/wfs/src/main/java/org/geoserver/wfs/WFSGetFeatureOutputFormat.java

```diff
@@ -217,6 +217,34 @@ protected int getNumDecimals(List featureCollections, GeoServer geoServer, Catal
         return numDecimals;
     }
 
+    /**
+     * Helper method that checks if coordinates measured values should be encoded for the provided
+     * feature collections.
+     *
+     * @param featureCollections features collections
+     * @param catalog GeoServer catalog
+     * @return TRUE if coordinates measures should be encoded, otherwise FALSE
+     */
+    protected boolean encodeCoordinatesMeasures(List featureCollections, Catalog catalog) {
+        boolean encodeCoordinatesMeasures = true;
```

Comment:
This bit is also affected by the defaults, in case we decided to flip them after review